### ISSUE TITLE
What's the final loss ?

### ISSUE THREAD

**Issue by PigletOS** [github-luffy/PFLD_68points_Pytorch#13]
What is the final loss you get? I tried to reimplement PFLD using keras. When I was training, the loss saturated around 30 at the early beginning and the loss was still around 30 after 20k iterations.

**Comment by github-luffy** [comment]
Still use the loss idea in the PFLD paper, but replace L1Loss with wing loss

**Comment by epoc88** [comment]
Same here, I think there is loss calculation issues, it is always around 30.
see this

https://github.com/github-luffy/PFLD_68points_Pytorch/issues/26
